iModelCore/GeomLibs/geom/test/CurvePrimitiveTest/t_CurvePrimitive.cpp

```diff
@@ -280,6 +280,152 @@ TEST(CurveVectorA, PointIsPlanar)
     CheckPlanarity (*curves, DPoint3d::From (0,0,0), DPoint3d::From (5,2,-3), 0.2);
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+    TEST(CurveVectorA, InitFromPointsOnAxisAndArc)
```

Comment:
1. Indentation is off throughout this test.
2. I'd verify that the returned arc.startPoint and input start point are the same when theta0 is absent or 2n\pi.
3. Similarly, I'd verify that they are not the same when theta0 is not 2n\pi.